src/components/Chat/InputMessageV2.tsx

```diff
@@ -426,7 +426,7 @@ const InputMessageV2 = ({
         {isShowToggle && (
           <ToggleSwitch
             labelLeft={
-              !inputFocus ? (isAnonimityEnabled ? 'Anonymity' : 'Anonymity disabled') : undefined
+              !inputFocus ? (isAnonimityEnabled ? 'Incognito' : 'Incognito disabled') : undefined
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from 'Anonymity' to 'Incognito' is a good one for user interface clarity. However, it would be better to avoid hardcoding these strings directly in the component. Instead, consider using a constants file or localization/internationalization library to manage all your application's text. This will make it easier to maintain and update the text, as well as support multiple languages in the future.
```diff
- !inputFocus ? (isAnonimityEnabled ? 'Incognito' : 'Incognito disabled') : undefined
+ !inputFocus ? (isAnonimityEnabled ? CONSTANTS.INCOGNITO : CONSTANTS.INCOGNITO_DISABLED) : undefined
```
In the above code, `CONSTANTS` is an object that contains all the constant values used across your application. You can define it in a separate file and import it wherever needed.


<!-- This is an auto-generated comment by OSS CodeRabbit -->